docs/reference/api/platform/Error.md

```diff
@@ -20,6 +150,132 @@ void *operator new(std::size_t count) {
 }
 ```
 
+#### Using `MBED_ERROR()` function with module information
+
+The code below uses an MBED_ERROR function to print a fatal error indicating an out-of-memory condition with module name specified as MODULE_APPLCIATION.
+
+```C
+void receive_data(unsigned char *buffer) {
+    if (NULL == buffer) {
+        MBED_ERROR1(MBED_MAKE_ERROR(MBED_MODULE_APPLICATION, MBED_ERROR_CODE_INVALID_ARGUMENT), "Buffer pointer is Null", 0 );
+    }
+    // read the data into given buffer
+    
+}
+```
+
+#### Using `MBED_WARNING()` function with module information and return with Mbed Error status
+
+The code below uses an MBED_WARNING function to report a invalid configuration attempt with module name specified as MBED_MODULE_PLATFORM.
+
+```C
+mbed_error_status_t configure(int config_value) {
+    if (config_value > 10) {
+        //Log the fact that a invalid configuration attempt was made and return with error code
+        MBED_WARNING1(MBED_MAKE_ERROR(MBED_MODULE_PLATFORM, MBED_ERROR_CODE_UNSUPPORTED), "Buffer pointer is Null", 0 );
```

Comment:
Is the error text here accurate? Something like `..., "Invalid config parameter", config_value);` or something looks closer.

Reply:
That was a mistake, will fix it.